### PR TITLE
provided method to fetch editorial articles

### PR DESCRIPTION
**Pull request by wsromek**
@filipgolonka @rafalkanski

### CODE REVIEW

File: docs/USAGE.md [hunk 1]
```diff
@@ -39,6 +39,20 @@ try {
 }
 ```
 
+### Fetching one editorial article by id for publication id
```
Comment (filipgolonka) [inline docs/USAGE.md:42]: add explanation for `editorial` - it is not so intuitive. Maybe sth like `editorial (with status different than published)`.

**Comment by rafalkanski** [comment]
👍

**Review by rafalkanski — APPROVED** [review]

File: src/Snt/Capi/Repository/Article/FindEditorialParameters.php [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+use Snt\Capi\Http\ApiHttpPathAndQuery;
+use Snt\Capi\Repository\FindParametersInterface;
+
+final class FindEditorialParameters implements FindParametersInterface
+{
+    const ARTICLES_PATH_PATTERN = 'publication/%s/editorials/%s';
+
+    /**
+     * @var string
+     */
```
Comment (filipgolonka) [inline src/Snt/Capi/Repository/Article/FindEditorialParameters.php]: do you need it?

File: src/Snt/Capi/Repository/Article/FindEditorialParameters.php [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+    private $publicationId;
+
+    /**
+     * @var int|null
+     */
```
Comment (filipgolonka) [inline src/Snt/Capi/Repository/Article/FindEditorialParameters.php]: as above

**Review by filipgolonka — APPROVED** [review]

File: src/Snt/Capi/Repository/Article/FindEditorialParameters.php [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+    /**
+     * @return int
+     */
+    public function getArticleId()
+    {
+        return $this->articleId;
+    }
+
+    /**
+     * {@inheritdoc}
```
Comment (rafalkanski) [inline src/Snt/Capi/Repository/Article/FindEditorialParameters.php]: this line can be removed i think

**Pull request merged**